### ISSUE TITLE
a bug about parseFilter

### ISSUE THREAD

**Issue by marzk** [mysticatea/vue-eslint-parser#37]
try `eslint` to reproduce the bug.

```vue

<template>
  <div>{{ 1 | identity() || 0 }}</div>
</template>

<script>
export default {};
</script>

```

This is because `identity() || 0` is `LogicalExpression` but not `CallExpression`.

This is the bug code.
```javascript

//  `parseFilter` in `./src/script/index.ts`

if (argsCode != null) {
    const { ast } = parseScriptFragment(
        `0${argsCode}`,
        locationCalculator.getSubCalculatorAfter(paren - 1),
        parserOptions,
    )
    const statement = ast.body[0] as ESLintExpressionStatement
    const callExpression = statement.expression as ESLintCallExpression

    ast.tokens!.shift()
    
    // this is LogicalExpression
    for (const argument of callExpression.arguments) {
        argument.parent = expression
        expression.arguments.push(argument)
    }
    
    // ...
}

```

im not good at parser for now, and I dont know how to fix this bug. So i only create an issue but not a pr.

**Comment by mysticatea** [comment]
Thank you for this report.

It doesn't look a documented behavior (https://vuejs.org/v2/guide/filters.html). Would you explain how it works?

cc: @yyx990803 @michalsnik

**Comment by mysticatea** [comment]
You are right. We should fix this.

@yyx990803 I'd like to ask the specification. Should we go which way?

1. ESLint allows that filters usage.
1. ESLint reports it as a syntax error.

**Comment by yyx990803** [comment]
I think this should be a syntax error.

**Comment by mysticatea** [comment]
@yyx990803 Thank you!
